pkgs/by-name/ra/redbean/package.nix

```
@@ -0,0 +1,44 @@
+{ lib, stdenv, fetchurl }:
+
+let
+  libPath = lib.makeLibraryPath [ stdenv.cc.cc  ];
```

Looks like you haven't run a formatter.
```suggestion
  libPath = lib.makeLibraryPath [ stdenv.cc.cc ];
```

Moreover, where is this variable even used?